soapboxpb/environment.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package soapbox;
+option go_package = "soapboxpb";
+
+import "application.proto";
+
+service Environments {
+    rpc ListEnvironments (ListEnvironmentRequest) returns (ListEnvironmentResponse) {}
+    rpc GetEnvironment (GetEnvironmentRequest) returns (GetEnvironmentResponse) {}
+    rpc CreateEnvironment (CreateEnvironmentRequest) returns (CreateEnvironmentResponse) {}
+    rpc DestroyEnvironment (DestroyEnvironmentRequest) returns (DestroyEnvironmentResponse) {}
+    rpc CopyEnvironment (CopyEnvironmentRequest) returns (CopyEnvironmentResponse) {}
+}
+
+message ListEnvironmentRequest {}
+
+message EnvironmentSummary {
+    int32 id = 1;
+    string name = 2;
+}
+
+message ListEnvironmentResponse {
+    repeated EnvironmentSummary environments = 1;
```

Comment:
It would. But each app would have on the order of single-digit environments (probably 2 or 3 at most), and most apps have on the order of tens of environment variable pairs (a dozen or so at most). So the on-the-wire payload even if all environments for an app are included is going to be on the order of single-digit kilobytes.